### ISSUE TITLE
Core compatability with some TVs

### ISSUE THREAD

**Issue by CollectorVision** [CollectorVision/Phoenix-Colecovision#1]
Some TVs do not like the current 25MHz pixel clock used with the CV core. We are working to fix the core to move to 720x480p with 27Mhz pixel clock (which is what the 2600 core is using).

**Comment by ikrananka** [comment]
This also seems to be a potential issue with computer monitors as well.  I've been testing my Phoenix with a Dell U2410 monitor and am experiencing some odd behaviour, such as blank screens and the red light staying on for a while when turning the Phoenix off.  I'll do some more testing and try to more accurately document my observations.

**Comment by lilalien** [comment]
I could not get a picture at all to display on my 2017 Sony Bravia 40" display. I tried multiple HDMI ports on the TV, as well as several different cables. Connected it to my secondary display in my game room which was a 20" Vizio and it displays perfectly.

Perhaps I'll test an HDMI switch I have to see if that makes a difference on the 40"

**Comment by KingofGrills** [comment]
I have 2 Samsung 4K TVs.  Neither TV recognizes the signal from the Phoenix.  I have tried connecting directly to the HDMI ports and also through HDMI switchers.

Please fix this issue soon.

**Comment by Atarileaf** [comment]
Could this bug also manifest itself with audio distortion such as I'm experiencing?

**Comment by lilalien** [comment]
I also have the problem. My Sony Bravia from 2017 will not display a picture I only hear sound. My spare very old Vizio from 8 years ago shows video and sound.

**Comment by CollectorVision** [comment]
Some users have had success using this adapter to make the system work on TVs that it previously wouldn't.

https://www.amazon.com/dp/B01N06Q9WH/ref=cm_sw_r_other_apa_i_aCzWDb2K5MSPS?fbclid=IwAR3_FS451QnGLA7EVdREomQLkzTvgcq9UikPLUqqfXiqS_6M7jkYN8FFOoc

**Comment by lilalien** [comment]
I've had success now on both a old Vizio LCD and now a 4 year old 46" inch Samsung TV in my master bedroom.. Still no luck on the 2 year old 40" Sony Bravia.

I'm hoping tomorrow to try out a powered HDMI switch I have spare. I'll post my results.

**Comment by lilalien** [comment]
So I did try out a powered HDMI switch I had, and no luck. Still no picture on the Sony Bravia 40" in my game room I was hoping to use this with. So I have to use the Phoenix with the spare 20" Vizio I have until this gets fixed...

**Comment by wongojack** [comment]
I cannot get the Phoenix to display at all on my Vizio E421VA from 2010 (invalid format)

**Comment by lilalien** [comment]
How is the research on this progressing?

I'd really like to use this on my main game room TV (2017 Sony Bravia 40")

**Comment by lilalien** [comment]
Anyone out there? :)

**Comment by CollectorVision** [comment]
We are still here and working on it. No new ETA on when it will be ready.

**Comment by lilalien** [comment]
Thank you very much! Looking forward to it!

**Comment by lilalien** [comment]
Hey,

So I'm hoping this is a priority, especially before sending out a new batch?

I've had the unit 4 months with this issue, and I would really like to see a resolution to this which is my only major issue with the unit.

The HDMI switch I was going to use as a temporary fix did not fix the problem, I still get the same issue on my Sony LCD's in my game room (two different models). I want to stop using the unit on a smaller Samsung tv in my bedroom and actually use the unit in my game room with everything else.

Please consider doubling your efforts on this fix. If I knew it wouldn't work on my TV I would never have purchased it.

**Comment by Atarileaf** [comment]
I don't think theres been much action on any of the tv issues afaik

**Comment by lilalien** [comment]
> I don't think theres been much action on any of the tv issues afaik

Yeah, which is very sad.. Just because we stop asking doesn't mean it isn't important. This was never brought to attention of those who ordered it. While I love the unit otherwise, and reported bugs regularly early on it is very unfortunate this clock issue hasn't been a priority before moving to other things.

I no longer recommend it on social media until this is resolved. I have quite a lot of youtube and facebook group followers that I would LOVE to recommend a purchase to. Please resolve this.

**Comment by CollectorVision** [comment]
> Hey,
> 
> So I'm hoping this is a priority, especially before sending out a new batch?
> 
> I've had the unit 4 months with this issue, and I would really like to see a resolution to this which is my only major issue with the unit.
> 
> The HDMI switch I was going to use as a temporary fix did not fix the problem, I still get the same issue on my Sony LCD's in my game room (two different models). I want to stop using the unit on a smaller Samsung tv in my bedroom and actually use the unit in my game room with everything else.
> 
> Please consider doubling your efforts on this fix. If I knew it wouldn't work on my TV I would never have purchased it.

The HDMI switch only solves issue #4 not this particular issue. We are working on this issue. The change to make this work is not trivial and is a massive change in the core to make this work. This is the top priority at the moment.

**Comment by lilalien** [comment]
> > Hey,
> > So I'm hoping this is a priority, especially before sending out a new batch?
> > I've had the unit 4 months with this issue, and I would really like to see a resolution to this which is my only major issue with the unit.
> > The HDMI switch I was going to use as a temporary fix did not fix the problem, I still get the same issue on my Sony LCD's in my game room (two different models). I want to stop using the unit on a smaller Samsung tv in my bedroom and actually use the unit in my game room with everything else.
> > Please consider doubling your efforts on this fix. If I knew it wouldn't work on my TV I would never have purchased it.
> 
> The HDMI switch only solves issue #4 not this particular issue. We are working on this issue. The change to make this work is not trivial and is a massive change in the core to make this work. This is the top priority at the moment.

Thank you so much for the update, I am looking forward to playing this on the big screen soon.

I am totally not trying to be a nag, just really do appreciate what you guys are doing and want to see this product be the best it can be. I hope you and your family are healthy and safe.

**Comment by Atarileaf** [comment]
Got a Samsung 50 inch plasma from a friend. Mode not supported error so I have 2 TVs that dont work with the Phoenix. Is there any good news on a fix? Last update was March

**Comment by lilalien** [comment]
I've had it work on my Samsung, but not on my Sony in my game room.. I haven't tried it on my other Samsung, but its packed up at the moment readying for a move. Hope to see this fix soon, its very important to me!

**Comment by CollectorVision** [comment]
This issue is still being worked on and we are hoping to have it finished before we ship the next batch of systems, but we do not have a firm date for a fix yet.

**Comment by lilalien** [comment]
Very much appreciated! Thank you!

**Comment by lilalien** [comment]
I hope you're nailing this bug to the wall! I'd like to switch out the way I'm currently playing it...

**Comment by CollectorVision** [comment]
This adapter seems to work for me with little noticeable lag for HDMI in and HDMI out.

https://www.amazon.com/gp/product/B07KVRYWDF/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1&fbclid=IwAR3aOmJ4kHvpCSFnk3T81CL50RvnfY96thHYOBp4Tzvk6Y7ycyXlfZ3wMDQ

**Comment by wongojack** [comment]
Thank you - HDMI to HDMI is interesting if it increases compatibility.  Is there anyway to get an analog signal out of the Phoenix?  Component out?

**Comment by lilalien** [comment]
Hey.. So how is this bug fix coming along? Significant progress?

**Comment by Mika73** [comment]
After I updated my projector to Xgimi Horizon Pro I couldn’t get picture of Phoenix anymore. Hope it’s possible update Phoenix to fix this problem.

**Comment by VerseData-EP** [comment]
Any updates on this? Come on folks?

**Comment by kelrodjr** [comment]
Come on guys... I know this isn't a trivial fix, but it's been over 3 years since this issue was identified.  I support what you guys do and love that you were able to bring the Phoenix to life, but it's beyond irritating that I can't play it on my own TV. As of right now my Phoenix is as useful as a paperweight ever since I bought it going on two years ago.  Please give us a status update, a timeline, SOMETHING.
